modules/kuberay-cluster/main.tf

```diff
@@ -116,6 +116,12 @@ resource "kubernetes_network_policy" "ray-job-and-dashboard-namespace-network-po
         protocol = "TCP"
       }
 
+      # Ray client
+      ports {
```

Comment:
do we need to include other ray cluster port as well?  metrics port for sure.. so gmp pods can poll metrics from this endpoint.

Reply:
All ports are allowed within the namespace and for the gke-gmp-system namespace under the `kuberay-cluster-allow-network-policy` policy below. I don't want to make this too granular since it's only an example.

Reply:
The intention of these firewall rules is to be configurable for end user to access the dashboard, client, and job submission ports.